src/Sylius/Component/Resource/Metadata/Metadata.php

```diff
@@ -148,6 +148,11 @@ public function hasParameter($name)
         return array_key_exists($name, $this->parameters);
     }
 
+    public function getParameters()
```

Comment:
I get the feeling that ultimately we should remove the "parameters" from the metadata and have more specific methods (e.g. getForm, getClass, etc. (see also #5296)).